### PR TITLE
arange item in both room and fixed css in leaderboard

### PR DESCRIPTION
**Pull request by harfrancois**

### CODE REVIEW

**Review by denekm — APPROVED** [review]

File: js/game.js [hunk 1]
```diff
@@ -74,14 +74,14 @@ function Inventory(pojoItems) {
...
-    // items.push(new Items('keyboard'));
-    // items.push(new Items('mouse'));
-    // this.items.push(new Items('flashlight'));
-    // items.push(new Items('backback'));
-    // items.push(new Items('textbooks'));
-    this.items.forEach(item => item.render());
+    this.items.push(new Items('logo', false, 'index.html', '30px', '5rem', genericClick));
+    this.items.push(new Items('laptop', false, 'index.html', '3px', '8rem', genericClick));
+    this.items.push(new Items('keyboard', false, 'classroom.html', '60px', '9rem', genericClick));
+    this.items.push(new Items('mouse', false, 'classroom.html', '100px', '5rem', genericClick));
```
Comment (laptopadventure) [inline js/game.js]: add the slash before the webpage url

File: js/game.js [hunk 2]
```diff
@@ -74,14 +74,14 @@ function Inventory(pojoItems) {
...
+    this.items.push(new Items('flashlight', false, '/index.html', '666px', '5rem', flashlightClick));
+    this.items.push(new Items('backback', false, '/index.html', '333px', '5rem', laptopClick));
+    this.items.push(new Items('textbooks', false, 'classroom.html', '555px', '5rem', genericClick));
```
Comment (laptopadventure) [inline js/game.js]: like so:
```suggestion
    this.items.push(new Items('textbooks', false, '/classroom.html', '555px', '5rem', genericClick));
```

File: js/game.js [hunk 3]
```diff
@@ -74,14 +74,14 @@ function Inventory(pojoItems) {
...
   } else {
     //first time setup, creates all items with their default vals
-    this.items.push(new Items('logo', false, '/index.html', '30px', '10rem', laptopClick));
-    // items.push(new Items('laptop'));
-    // items.push(new Items('keyboard'));
-    // items.push(new Items('mouse'));
-    // this.items.push(new Items('flashlight'));
-    // items.push(new Items('backback'));
-    // items.push(new Items('textbooks'));
-    this.items.forEach(item => item.render());
```
Comment (Dcastro99) [inline js/game.js:84]: add this back

File: js/game.js [hunk 4]
```diff
@@ -109,8 +109,9 @@ function Inventory(pojoItems) {
   };
   this.render();
 }
-
-
+function genericClick(){
+  this.render();
```
Comment (laptopadventure) [inline js/game.js]: this doesn't exist in this case but whatever this is temporary anyways

**Review by laptopadventure — CHANGES_REQUESTED** [review]

File: js/game.js [hunk 5]
```diff
@@ -74,14 +74,14 @@ function Inventory(pojoItems) {
...
-    // this.items.push(new Items('flashlight'));
-    // items.push(new Items('backback'));
-    // items.push(new Items('textbooks'));
-    this.items.forEach(item => item.render());
+    this.items.push(new Items('logo', false, 'index.html', '30px', '5rem', genericClick));
+    this.items.push(new Items('laptop', false, 'index.html', '3px', '8rem', genericClick));
+    this.items.push(new Items('keyboard', false, 'classroom.html', '60px', '9rem', genericClick));
+    this.items.push(new Items('mouse', false, 'classroom.html', '100px', '5rem', genericClick));
+    this.items.push(new Items('flashlight', false, '/index.html', '666px', '5rem', flashlightClick));
+    this.items.push(new Items('backback', false, '/index.html', '333px', '5rem', laptopClick));
```
Comment (Dcastro99) [inline js/game.js]: needs to be generickClick

File: js/game.js (hunk 4 above)
Comment (laptopadventure) [reply js/game.js]: eeeh replace with return

**Review by Dcastro99 — CHANGES_REQUESTED** [review]

**Review by Dcastro99 — APPROVED** [review]

**Review by laptopadventure — APPROVED** [review]

**Pull request merged**